src/js/components/IconTextButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { THEME } from "../constants";
+
+import IconButton from "./IconButton";
+
+const OuterButton = styled.button`
```

Comment:
Maybe just `Container` can we use flex for this and to control both children components when on hover

Reply:
Sure, I can rename this to `Container` and we can deal with the children components hover/click styling here